[FEAT]login11.HTML

resader1 opened the pull request:
**Check List :memo:**
- [x] Pull Request 제목 : [유형] 변경 사항 요약
- [x] 설명 & 이슈 번호 작성

**설명 (필수)**

이 PR에서 어떤 변경을 했는지 상세히 설명해주세요.
관련된 Issue 번호를 포함하세요.

1. #24 

2. 변경 사항
헤더푸터
글씨체 색상
인풋,버튼 설정 최신화
사이드바 삭제


4. 스크린샷 (선택)
![가입완](https://github.com/user-attachments/assets/06983891-24af-42ab-8930-8e5a05deecad)
![로그인](https://github.com/user-attachments/assets/568728e4-2389-48d3-8478-a2ffab26b6a8)
![아디찾기](https://github.com/user-attachments/assets/588cdef7-159e-44ed-9270-94d2042c78d3)
![약관동의](https://github.com/user-attachments/assets/9d4f2a8e-691f-4977-abd7-952fb3c29218)
![이메일인증](https://github.com/user-attachments/assets/2af2fc45-59fa-4432-8dac-a1a8ab1b0081)
![정보입력](https://github.com/user-attachments/assets/9a0996ce-4d18-42ef-b3cd-d25ef3fca6f2)


<br>

---

**review 참고 사항 (선택)**

추가로 알릴 내용이 있다면 여기에 작성해주세요.

1.

2.

resader1 committed:
전 페이지 작업 1차 완료

헤더푸터
글씨체 색상
인풋,버튼 설정 최신화
사이드바 삭제

resader1 committed:
0627 0621 수정

radius 전 항목 7px 수정
버튼 width 누락 수정(execpt input)

resader1 committed:
p 수정

p 중복으로 인한 색 변경 확인/수정 완료

css 전체적 수정 중

resader1 committed:
버튼 수정

버튼 공통으로 전부 수정
input창 설정 변경

resader1 committed:
Update find1.html

resader1 committed:
co

co

resader1 committed:
F

resader1 committed:
join3(회원정보입력) 변경

스타일 변경
비밀번호 8자 이상 입력하게 변경
비밀번호 일치 여부에 따라 border 색 변경되며 문구 뜨게 변경

kb4954 commented on login/join31.html:
```diff
@@ -617,184 +55,149 @@ <h4 class="fig-exchange-book">교환 독서</h4>
+                                    <span>*</span>
+                                </th>
+                                <td>
+                                  <input type="password" name="pw" class="pw" placeholder="영문,숫자,특수문자 사용 8자 이상." >
```
![image](https://github.com/user-attachments/assets/fb896822-194a-4027-8a60-c38f7ea7f600)
여기 글자랑 input박스 사이에 공간 넣어주세용..!!! margin이나 gap 넣어주세요

resader1 committed:
30일 수정

join3 출력문구 margin값 조정
join2 수정

resader1 committed:
Update join21.html

resader1 committed:
Update join21.html

resader1 committed:
Update join21.html

resader1 committed:
join2 css분리

약관창 넓힘, section으로 구분 후 전체 form으로 덮음

resader1 committed:
Update join21.html

resader1 committed:
간격,크기

넓히고 키움

resader1 committed:
Update login11.html

null4848 reviewed (commented):
고생하셨어요!
나머지 기능 부분은 나중에 추가해주시면 될 것 가타용

resader1 committed:
기능추가

이용약관 - 체크박스 전부 체크되어있어야 페이지 넘어갈 수 있도록 변경
정보입력 - 이메일 선택시 자동입력 및 수정 불가

resader1 committed:
코멘트 수정

지적사항 수정

null4848 reviewed (commented):
폰트 색상도 확인 부탁드려요!

resader1 committed:
수정

다함

resader1 commented on login/join41.html:
```diff
@@ -0,0 +1,138 @@
+            <br>
+            <p style="text-align:center; font-size:20px; font-weight:5px;">가입해주셔서 감사합니다.</p>
+          </div>
+            <br><br><br><br><br><br><br><br><br><br><br><br>
```
지적하신부분 수정해서 올렸습니다.

resader1 committed:
radius 수정

약관 내부 부분 radius 삭제

resader1 committed:
join3간격조정

verticalalign:top이 안 먹혀서 전체적으로 tr을 verticalalign:middle로 맞춤.
관심분야는 middle일 시 괴리감 들어 top으로 설정

resader1 committed:
조정

버튼 조정 마진값 조정

resader1 committed:
페이지 테스트 중

copy본은 결정되면 삭제 예정

resader1 committed:
페이지 수정

로그인 페이지 - 아이디/비밀번호 찾기 변경

resader1 committed:
Delete Share_AddGroup1.css

resader1 committed:
ㄹ

hwk3005 commented:
회원가입 - 이용약관 html 코멘트 드립니다!  (아래와 같이 수정되면 좋을 것 같아서 참고해서 수정해주시면 될 것 같습니다.)

- 수정 전
![image](https://github.com/user-attachments/assets/da7b747f-c8ab-401d-81ca-2ce811071d11)

- 수정 후
![image](https://github.com/user-attachments/assets/5b8ba843-13db-468d-877a-49fef665537f)

1. 회원가입-이용약관
  1) 약관별 큰 틀
.contract {
    width: 1137px;
    margin: 20px auto 80px;
    padding: 30px 40px 35px;
    border-radius: 7px;
    background-color: #f5f5f5;
    /* border: 1px solid #2D7868; */
    box-shadow: 0 1px 4px #ccc;
}
  2) 약관 h1
.contract h1 {
    text-align: center;
    color: #1a1a1a;
    margin-bottom: 30px;
    font-size: 25px;
    font-weight: 700;
}
  3) 약관 text
.contract .text {
    width: 95%;
    height: 230px;
    background-color: #fafafa;
    padding: 10px 45px;
    white-space: pre-line;
    overflow-y: scroll;
    margin: 0 auto 30px;
    box-shadow: 0 2px 8px #A0A0A0;
    line-height: 1.8;
}
  4) 하단 약관 체크박스
.contract .yes p {
    /* color: #2D7868; */
}

resader1 committed:
Update join3.css

resader1 committed:
Update join2.css

hwk3005 reviewed (dismissed):
회원가입-이용약관 코멘트 적용된 거 확인

resader1 committed:
deq

제출용 최종수정

resader1 committed:
Update find2.css
